pkg/kubelet/networks.go

```diff
@@ -54,6 +55,8 @@ func (nh *networkHost) SupportsLegacyFeatures() bool {
 // methods, because networkHost is slated for deletion.
 type criNetworkHost struct {
 	*networkHost
+	// criNetworkHost currently support legacy features. Hence no need to support PortMappingGetter
```

Comment:
I think the naming here is confusing. The criNetworkHost is not actually used by CRI shims. But the good old kubelet runtime.